### DISCUSSION TITLE
Problems with iminuit on ubuntu version 22.04.2 ?

### DISCUSSION THREAD

**Discussion by farrukhazfar** [scikit-hep/iminuit#851] [General]
Hi Folks, 
I am on ubuntu 22.04.2 and am using python3 version 3.10.6. Last year everything I am about to describe was working fine
Unfortunately it seems that I am missing the full functionality of Minuit in iminuit
So here goes : 

python3
>>> from iminuit import Minuit
>>> import numpy as np
>>> def LSQ(a, b):
...     return np.sum((y - line(x, a, b)) ** 2 / sigma_y ** 2)

Next I try to setup the parameters with initial values and fix b
>>> m = Minuit(LSQ, a=0, b=0.4, fix_b=0.4)

and I get the following error
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/azfarl/.local/lib/python3.10/site-packages/iminuit/minuit.py", line 629, in __init__
    self._init_state = _make_init_state(self._pos2var, start, kwds)
  File "/home/azfarl/.local/lib/python3.10/site-packages/iminuit/minuit.py", line 2491, in _make_init_state
    raise RuntimeError(
RuntimeError: fix_b is not one of the parameters [a b]

I am having the same issue trying to use limit_a or limit_b etc - wondering what is going on - any help will be appreciated

best wishes
Farrukh

**Comment by HDembinski** [comment]
This has been answered in the issues https://github.com/scikit-hep/iminuit/issues/852
